zenith_utils/src/auth.rs

```diff
@@ -113,6 +112,14 @@ impl JwtAuth {
     }
 }
 
+impl std::fmt::Debug for JwtAuth {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("JwtAuth")
+            .field("validation", &self.validation)
+            .finish()
+    }
+}
+
```

Comment:
If I understand correctly, the idea is to hide the 'decoding_key' from debugging format. That's sensible. This change is currently part of the "bump pageversion dependencies" commit, was this required by changes in dependencies? This seems like a good idea on its own, so maybe extract it to a separate commit.

Reply:
Sure.

Initially that came as a preparation for new `jsonwebtoken` library version update, that turned the `DecodingKey` into `!Debug`, but indeed this is not obvious, extracted.